### PR TITLE
Change go package tool from dep to go mod.

### PR DESCRIPTION
**Pull request by laodouya**

### CODE REVIEW

File: docker/builder/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:alpine3.9
+FROM golang:alpine3.10
```
Comment (leventeliu) [inline docker/builder/Dockerfile:1]: Could we just use `latest` tag, if no version-specified feature is actually needed?

Comment (laodouya) [reply docker/builder/Dockerfile:1]: latest tag is based on buster, not alpine.

**Comment by auxten** [comment]
LGTM

**Review by auxten — APPROVED** [review]

**Review by leventeliu — APPROVED** [review]
LGTM.

**Pull request merged**